geocachingapi/models.py

```diff
@@ -72,8 +100,10 @@ def update_from_dict(self, data: Dict[str, Any]) -> None:
 
         self.tracking_number = try_get_from_dict(data, "trackingNumber", self.tracking_number)
         self.kilometers_traveled = try_get_from_dict(data, "kilometersTraveled", self.kilometers_traveled)
+        self.miles_traveled = try_get_from_dict(data, "milesTraveled", self.miles_traveled)
         self.current_geocache_code = try_get_from_dict(data, "currectGeocacheCode", self.current_geocache_code)
         self.current_geocache_name = try_get_from_dict(data, "currentGeocacheName", self.current_geocache_name)
+        self.is_missing = try_get_from_dict(data, "isMissing", self.current_geocache_name)
```

Comment:
Is self.current_geocache_name right here?

Reply:
Nope, good spot! Fixed it!